### PR TITLE
Fix syntax error in libretro port

### PR DESCRIPTION
**Pull request by guoyunhe**
Array syntax error. `,` should be placed after each element.

### CODE REVIEW

**Comment by Ghabry** [comment]
Oh. We forgot to test libretro before merging the PR :/. Sorry.

Could you just remove the leading comma on line 30 and keep it as is. Thanks!

There was a reason for the leading commas:

>>
> Ghabry: Is there a reason for using leading commas?
>
> fmatthew5876: Yes, with leading commas you don't need to worry about the last element being different. This makes it easier handling all these #ifdef

**Comment by guoyunhe** [comment]
For modern C++ compilers, the last comma will always be ignored.

You can check the code of RetroArch for reference https://github.com/libretro/RetroArch/blob/master/retroarch.c#L411

**Comment by fmatthew5876** [comment]
He's right this was was my mistake we don't need to do this anymore.

We should have all our files consistent.

@guoyunhe Would you mind also applying this change to `input_buttons_desktop.cpp` and all the `platform/*/*_input+buttons.cpp`?

Thanks for the patch!

**Comment by Ghabry** [comment]
Well I don't want to outsource the work of cleaning up our code base to external contributors.
I can push a commit to the PR to fix this.
Thx for the contribution and for moving to the upstream repo for libretto opensuse :)

**Comment by Ghabry** [comment]
Jenkins: test this please

File: src/platform/psp/psp_input_buttons.cpp [hunk 1]
```diff
@@ -23,19 +23,19 @@
 
 Input::ButtonMappingArray Input::GetDefaultButtonMappings() {
 	return {
-		,{DECISION, Keys::JOY_1} // Circle
```
Comment (Ghabry) [inline src/platform/psp/psp_input_buttons.cpp:26]: ^-- same syntax error as in libretro input

**Pull request merged**